src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,17 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.14.4
+
+This is a bugfix release:
+
+* Fix stale IPtable rules along with globalIP leak which can sometimes happen as part of stress testing.
+* Handle out of order remote endpoint notifications properly in various Route Agent handlers.
+* Ensure that reverse path filtering setting is properly applied on the `vx-submariner` and `vxlan-tunnel` interfaces after they are created.
+  This fix was necessary for RHEL9 nodes where the setting was sometimes getting overwritten.
+* Fix issues while spawning Gateway nodes during cloud prepare for clusters deployed on OpenStack environment running OVN-Kubernetes CNI.
```

Comment:
I know there is some active discussion whether to use "Fix ..." or "Fixed ...", but in case you had not seen, the Release note PR used the following text - https://github.com/submariner-io/submariner/pull/2480/files